README.rst

```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
                                       --instance-type ml.m4.xlarge \
                                       --tag 1.3.0-cpu-py3
 
+Amazon Elastic Inference with MXNet in SageMaker
```

Comment:
I'd move this to a subsection under "Building Images"

Reply:
I don't agree because this is about using MXNet with Elastic Inference in SageMaker and not necessarily how to build the image.

Reply:
how to use MXNet with EI (I assume you're referring to the last section) belongs in the Python SDK README, not this repo's.

Reply:
now that the other section has been moved to the Python SDK repository, I still think this should go under "building". I'd at least move this to be above running tests; it doesn't make sense for this README to talk about building images then testing images then building more images